tests/30rooms/11leaving.pl

```diff
@@ -0,0 +1,299 @@
+multi_test "Setup a room, and have the first user leave (SPEC-216)",
+
+    requires => [qw(
+        make_test_room change_room_powerlevels do_request_json_for user
+        more_users can_create_room
```

Comment:
Put `can` requirements on their own line so it's easier to add new (valued) dependencies later.